yogstation/code/modules/mob/living/simple_animal/friendly/goats.dm

```diff
@@ -0,0 +1,347 @@
+/mob/living/simple_animal/hostile/retaliate/goat/clown
+	name = "Gary the Goat"
+	desc = "Rather kick your butt then tell jokes."
+	icon = 'yogstation/icons/mob/clownpets.dmi'
+	icon_state = "clown_goat"
+	icon_living = "clown_goat"
+	icon_dead = "clown_goat_dead"
+	gold_core_spawnable = NO_SPAWN
+	butcher_results = list(/obj/item/reagent_containers/food/snacks/meat/slab = 3, /obj/item/clothing/mask/gas/clown_hat = 1, /obj/item/clothing/head/yogs/goatpelt = 1)
+
+/mob/living/simple_animal/hostile/retaliate/goat/ras
+	name = "Ralsei Goat"
+	desc = "It just wants to give you a hug!"
+	icon = 'yogstation/icons/mob/goats/ras_goat.dmi'
+	icon_state = "rasgoat"
+	icon_living = "rasgoat"
+	icon_dead = "rasgoat_dead"
+	attacktext = "''hugs''"
+	gold_core_spawnable = NO_SPAWN
+	butcher_results = list(/obj/item/reagent_containers/food/snacks/meat/slab = 3, /obj/item/toy/plush/goatplushie/angry = 1, /obj/item/clothing/head/yogs/goatpelt = 1)
+
+/mob/living/simple_animal/hostile/retaliate/goat/blue
+	name = "Blue Goat"
+	desc = "Im blue da da di da da bahhhh."
+	icon = 'yogstation/icons/mob/goats/blue_goat.dmi'
+	icon_state = "bluegoat"
+	icon_living = "bluegoat"
+	icon_dead = "bluegoat_dead"
+	gold_core_spawnable = NO_SPAWN
+
+/mob/living/simple_animal/hostile/retaliate/goat/chocolate
+	name = "Chocolate Goat"
+	desc = "Actually just a goat with dark brown fur but I can see why you would think its made of chocolate though."
+	icon = 'yogstation/icons/mob/goats/chocolate_goat.dmi'
+	icon_state = "chocolategoat"
+	icon_living = "chocolategoat"
+	icon_dead = "chocolategoat_dead"
+	gold_core_spawnable = NO_SPAWN
+
+/mob/living/simple_animal/hostile/retaliate/goat/christmas
```

Comment:
remind me to make this replace pete during christmas

Reply:
if I remember sure lol